src/PerformanceTests/Transport.V7.MQMQ_v1/MsmqProfile.cs

```diff
@@ -14,7 +14,7 @@ class MsmqProfile : IProfile, INeedPermutation
     public void Configure(EndpointConfiguration endpointConfiguration)
     {
         var transport = endpointConfiguration.UseTransport<MsmqTransport>();
-        transport.ConnectionString(ConfigurationHelper.GetConnectionString("MSMQ"));
+        transport.DisableDeadLetterQueueing();
```

Comment:
Same as the RabbitMQ one, if this is hardcoded. Shall we make it fixed in previous majors too?